### DISCUSSION TITLE
[3 luglio 2024] Automi

### DISCUSSION THREAD

**Discussion by Exyss** [sapienzastudentsnetwork/automi-calcolabilita-complessita#31] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/457d92a1-e417-416f-9f5c-15159f9ef1dd)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/user-attachments/assets/6d8a9c83-4899-4579-855f-fa93992a9f91)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by daromacarlo** [comment]
<img width="1644" height="654" alt="immagine" src="https://github.com/user-attachments/assets/b8abbe0a-9f0c-4aae-b0b7-5d9cbcd8a205" />
